Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -98,9 +99,15 @@ script:
         } else if ((typeof data) ==='object') {
             extractFromData(data);
         }
+        //May be empty elements which should be filtred
+        queryResults = queryResults.filter(function (element) {
+            return element != undefined || element != null;
+        });
 
-        if(retVal.md.length > 0) {
-            var hr = tblToMd(command, retVal.md,'', '<br>');
+        if(queryResults.length > 0) {
+            retVal.md = queryResults;
+            retVal.context.NetWitness.Events = queryResults;
```

Comment:
`groupObject` will not always be an event, you use `buildHrAndContext` also to parse `netwitness-search` results too, and those are not events. Events should be only those who return from `netwitness-query`